doc/xml/daps_user_output.xml

```diff
@@ -277,4 +277,34 @@
    </para>
   </important>
  </sect1>
-</chapter>
+ <sect1 id="sec.daps.user.output.timestamp">
+ <title>Putting the actual date and time into the target document</title>
+ <para>
+ In order to get information about the build date and time, you can use the dbtimestamp tag.
```

Comment:
"In order to" can be removed. Just start with "To get...". Same with "you can..."

```
To get ..., use ...
```